marc_to_solr/lib/princeton_marc.rb

```diff
@@ -641,10 +641,18 @@ def alma_code_start_53?(code)
   code.to_s.start_with?("53") && code.to_s.end_with?("06421")
 end
 
+def scsb_code_start?(code)
```

Comment:
I had to add this filter of we ended up getting another call number for https://catalog.princeton.edu/catalog/99106471643506421, which is @sample41 in the tests.